hikari/files.py

```diff
@@ -473,6 +473,38 @@ async def read(
         async with self.stream(executor=executor) as reader:
             return await reader.read()
 
+    async def save(
```

Comment:
Maybe worth checking if the file exists before writing to it and erroring out if a `force` kwarg only arg is not set to True

Reply:
Yeah that's a good idea.